### PR TITLE
Kv readme closes #1300

### LINKED ISSUE #1300: Review WDL SPEC for accuracy
@geoffjentry has found a few inconsistencies in the spec (please add here), and there may be others lurking.  Let's give it a once over

### PR DESCRIPTION
**Pull request by katevoss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,7 +278,75 @@ Start a server on port 8000, the API for the server is described in the [REST AP
 
 # Getting Started with WDL
 
-For many examples on how to use WDL see [the WDL site](https://github.com/broadinstitute/wdl#getting-started-with-wdl)
+For many examples on how to use Workflow Description Language (WDL) see [the WDL website](https://software.broadinstitute.org/wdl/userguide/)
+
+## WDL Support
+
+:pig2: Cromwell :white_check_mark: supports the following subset of WDL features:
```
Comment (geoffjentry) [inline README.md]: I don't understand the check mark being there

**Comment by geoffjentry** [comment]
👍

**Comment by mcovarr** [comment]
:+1:

**Comment by katevoss** [comment]
@geoffjentry did you get a chance to look over the [spreadsheet of WDL features](https://docs.google.com/spreadsheets/d/1phWdti3sVNuthFmbIe8bSwyEuEb7r3EbFMxWzG2tgX4/edit#gid=0) a second time?

**Comment by geoffjentry** [comment]
@katevoss going to try to get to that today

**Comment by geoffjentry** [comment]
@katevoss I filled in all of the read_x and write_x blanks. Note that I tried many but not all of the permutations by hand for correctness, so it's possible that there are bugs in some of these but they're at least intended to be supported in those cases.

**Comment by katevoss** [comment]
I added the rest of the WDL features that Cromwell supports, list should be up to date now. 

But I have failing checks, is there something I can do about that, @ruchim ?

**Pull request merged**